schema-source/RS0001.schema.yaml

```diff
@@ -300,32 +300,32 @@ GridVariablesCooling:
       Data Type: "[Numeric][1..]"
       Units: "m3/s"
       Constraints: ">0.0"
-      Required: "if condenser_type = LIQUID"
+      Required: False
```

Comment:
For now, the only way around this is to make these not required for now. This is of course moot if we decide to move air-cooled and liquid cooled performance into separate maps.